pitopcli/display.py

```diff
@@ -0,0 +1,141 @@
+            elif self.args.decrement_brightness:
+                self.debug_print("REQ:\tDECREMENTING BRIGHTNESS", 1)
+                display.decrement_brightness()
+            elif vars(self.args).get('brightness_value') or self.args.brightness_value is not None:
+                self.debug_print("REQ:\tSETTING BRIGHTNESS TO " +
+                                 str(self.args.brightness_value), 1)
+                display.brightness = self.args.brightness_value
+            else:
+                self.debug_print("REQ:\tCURRENT BRIGHTNESS", 1)
+                print(display.brightness)
+
+        elif self.args.display_subcommand == "backlight":
+            if vars(self.args).get('backlight_value') is None:
+                self.debug_print("REQ:\tCURRENT BACKLIGHT", 1)
+                print(display.backlight)
+            else:
+                backlight_state = self.args.backlight_value > 0
+                self.debug_print(f"REQ:\tTURNING {'ON' if backlight_state else 'OFF'} BACKLIGHT", 1)
+                display.backlight = backlight_state
+
+        elif self.args.display_subcommand == "timeout":
+            if vars(self.args).get('timeout_value') is None:
+                self.debug_print("REQ:\tCURRENT TIMEOUT", 1)
+                print(display.blanking_timeout)
+            else:
+                self.debug_print(f"REQ:\tSETTING TIMEOUT TO {self.args.timeout_value}", 1)
+                display.blanking_timeout = self.args.timeout_value
+
+    @classmethod
+    def add_parser_arguments(cls, parser) -> None:
+        subparser = parser.add_subparsers(title="pi-top display utility",
+                                          description="Interface to communicate with the device\'s display",
+                                          required=True,
+                                          dest="display_subcommand")
+
+        # manage arguments common to subparser options
+        parent_parser = ArgumentParser(add_help=False)
+        parent_parser.add_argument("-v", "--verbose",
+                                   action="count")
+
+        # "pi-top display brightness"
+        brightness_parser = subparser.add_parser("brightness", help="Control display brightness", parents=[parent_parser])
+        cls.add_brightness_arguments(brightness_parser)
+
+        # "pi-top display backlight"
+        backlight_parser = subparser.add_parser("backlight", help="Control display backlight", parents=[parent_parser])
+        backlight_parser.add_argument("backlight_value",
+                                      help="Set the screen backlight state [0-1]",
+                                      type=int,
+                                      choices=range(2),
+                                      nargs='?')
+
+        # "pi-top display timeout"
+        timeout_parser = subparser.add_parser(
+            "timeout", help="Set the timeout before the screen blanks in seconds (0 to disable)", parents=[parent_parser])
```

Comment:
"timeout before screen blanks"
"screen blank time"
"screen blanking inactivity timeout"
etc. etc.

We should standardise text for screen blanking. Not sure 100% what this should be, so perhaps we should consult the team